wordpress/wp-content/plugins/memberful-wp/src/core-ext.php

```diff
@@ -21,17 +21,23 @@ function memberful_wp_nonce_field( $action ) {
  * @param array $content A set of websites that can be redirected to
  * @return array The $content plus Memberful domain
  */
-function memberful_wp_allowed_hosts( $content ) {
+function memberful_wp_allowed_hosts( $hosts ) {
   $site = get_option( 'memberful_site' );
+  $custom_domain = get_option( 'memberful_custom_domain' );
 
-  if ( !empty( $site ) ) {
-    $memberful_url = parse_url( $site );
+  if ( $site ) {
+    $components = parse_url( $site );
 
-    if ( $memberful_url !== false )
-      $content[] = $memberful_url['host'];
+    if ( $components ) {
+      $hosts[] = $components['host'];
```

Comment:
Non-blocking, but is there a difference between this and `array_push($hosts, $components['host'])`? I find the latter easier to understand.

Reply:
Answering my own question: looks like there isn't a difference: https://stackoverflow.com/questions/676677/how-to-add-elements-to-an-empty-array-in-php

Reply:
Good call 👍 dd91841